config/settings/base.py

```diff
@@ -327,7 +327,7 @@
     },
 }
 
-DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"django_datadog_logger\.middleware\.request_log"
+DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"django_datadog_logger\.middleware\.request_log|itou.*"
```

Comment:
je dirais qu'a minima faut corriger en ça, sinon on prend aussi `itouwhatever.module` et pas juste `itou.module` et on autorise `itou.` ce qui serais bizarre
```suggestion
DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"django_datadog_logger\.middleware\.request_log|itou\..+"
```
Sinon dans la doc il utilise cette forme donc ça pourrais aussi être une possibilité, car elle permet de gérer un peu tout les cas d'un coup et ça facilite l'ajout de nouveau module.
```suggestion
DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"^(django_datadog_logger|itou)(|\..+)$"
```
A voir si il y a une raison pour avoir cibler `django_datadog_logger.middleware.request_log` plutôt que `django_datadog_logger` mais au pire ceci devrais fonctionner :
```suggestion
DJANGO_DATADOG_LOGGER_EXTRA_INCLUDE = r"^(django_datadog_logger\.middleware\.request_log|itou)(|\..+)$"
```